core/slack_bot.py

```diff
@@ -0,0 +1,38 @@
+import os
+import logging
+from slackclient import SlackClient
+
+
+class SlackIntegration(object):
+    """Slack Integration class"""
+
+    def __init__(self):
+        """set the slack token"""
+        self.slack_client = SlackClient(os.getenv('SLACK_TOKEN'))
+
+    def get_user_slack_id(self, user_email):
+        """Get the slack user ID using the user email"""
+        response = self.slack_client.api_call("users.list")
+        users = response["members"]
```

Comment:
The tests are failing as a result of a `KeyError: 'members'` every time an `AssetStatus` object is saved in the database (there's a `post_save` receiver on that model) https://github.com/AndelaOSP/art-backend/blob/0d8e9a71b4cdae1c296ed0840f4e83f17f53addb/core/models/asset.py#L299-L310

One more thing to keep in mind is that the changes on this PR require an environment variable called `SLACK_TOKEN` to initialise the `SlackIntegration` class. Could be that this variable is not set on circleci.

Reply:
Also, update the [README](https://github.com/AndelaOSP/art-backend#set-up-environment-variables) to reflect the environment variables that need to be set.